user-guide/manual.adoc

```diff
@@ -442,6 +442,37 @@ Security Contexts operate in the context of a single Security Operation over a b
 
 An executable used to provide a test fixture and example BPA integration. However, the Mock BPA does not provide any of the normal processing required of a real BPA by <<RFC9171>>, it is limited to decoding and encoding BPv7 protocol data unit (PDU) byte strings, processing specific BPv7 primary block fields, providing BSL-required integration callbacks, and calling into the BSL for each bundle being processed at each interaction point. Users may reference the Mock BPA for an example of library and bundle workflow.
 
+[#sec-api-pp-json]
+==== JSON-Defined Policy Provider Rules
+The Mock BPA has the option to parse and load ION-DTN JSON policy rules, with some notable differences/modifications, into the sample policy provider. The current differences are as follows:
```

Comment:
Please include a reference to the ION documentation. Cite it here with `<<ref>>` and add it to the reference documents table in the frontmatter.